__tests__/functions.test.js

```diff
@@ -9,15 +9,15 @@ describe('Function Operators', () => {
     expect(exp(5)(3)).toBe(125)
   })
 
-  test('isNil -> should check if a value is null or undefined', () => {
+  /* test('isNil -> should check if a value is null or undefined', () => {
```

Comment:
Should we keep the `isNil` function or not? 🤔

Reply:
I don't see the benefit, so I've removed it.